controllers/submission.go

```diff
@@ -33,6 +33,23 @@ var fileNames = map[string]string{
 
 var cloudClient *http.Client
 
+func init() {
+	var err error
+	cloudClient, err = google.DefaultClient(context.Background())
```

Comment:
This is new, previously we had to pass a `context.Context` that came from `appengine.NewContext`.